python/lambda-from-container/app.py

```diff
@@ -42,9 +43,9 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
             ## Container Image.
             ## Pulled from the ECR repository.
             ##
-            ecr_image = aws_lambda.EcrImageCode(
+            ecr_image = typing.cast("aws_lambda.Code", aws_lambda.EcrImageCode(
```

Comment:
Provide explanation comment so that folks know why this is being done.

Reply:
Added comment for clarification